### PR TITLE
feat: Requirements process

### PR DESCRIPTION
**Pull request by brian-intel**
Adding initial template along with the requirements process.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## PR Checklist
+<!-- Please check if your PR fulfills the following requirements: -->
+
+- [ ] Added label to the Pull Request for easier discoverability and search
+- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
```
Comment (jim-wang-intel) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Is PR in this repo the same guideline as the one in main code?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Intel Retail Documentation Repository
```
Comment (NeethuES-intel) [inline README.md:1]: Will this repo only contain documentation ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+![Requirements Review Process Flow](./requirements-review-process-flow.png)
+
+## Requirements Process Breakdown
+
+- Submit a github issue and follow the requirements template.
+
+- Once a week there will be a requirements review meeting where we will approve, reject, or ask for modification to the requirement.
```
Comment (NeethuES-intel) [inline README.md:9]: should it be once a week or once every(#) week ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- If approved the requirement will be moved into the design stage in the documents project.
+
+- A detailed design document will be made by the submitter and development team
+
+- A pull request will be created and reviewed by the design review team. The design will be iterated until approved by the board.
+
+- Once approved a story will be created in the appropriate github project and prioritized.
```
Comment (NeethuES-intel) [inline README.md:17]: prioritized by team or PO ?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Requirements Process Breakdown
+
+- Submit a github issue and follow the requirements template.
+
+- Once a week there will be a requirements review meeting where we will approve, reject, or ask for modification to the requirement.
+
+- If approved the requirement will be moved into the design stage in the documents project.
+
+- A detailed design document will be made by the submitter and development team
```
Comment (NeethuES-intel) [inline README.md:13]: can we share any design document template to follow ? may be edgex one ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
```
Comment (NeethuES-intel) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: as this doc would be public, would be nice to have a checklist reminding the author that they are not sharing any Employer confidential details (specifically to gate Intel confidential details) eg
[ ] No Employer Confidential details are added

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (brian-intel) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: This PR template will be used for the design documents and yes we want to apply everything that makes sense from the main repos.

File: README.md (hunk 2 above)
Comment (brian-intel) [reply README.md:1]: Yes, only design documentation and other relevant docs.

File: README.md (hunk 3 above)
Comment (brian-intel) [reply README.md:9]: I'll change that. Currently we meet once a week but that can change depending on what works for us.

File: README.md (hunk 4 above)
Comment (brian-intel) [reply README.md:17]: This will be the PO.

File: README.md (hunk 5 above)
Comment (brian-intel) [reply README.md:13]: The template was roughly based on the EdgeX template but we can add/remove what makes sense.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jim-wang-intel) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: most likely if this repo is just documentation then probably only conventional commits `docs:` and `fix:` makes sense IMO

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (brian-intel) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Added

**Review by jim-wang-intel — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+![Requirements Review Process Flow](./requirements-review-process-flow.png)
+
+## Requirements Process Breakdown
+
+- Submit a github issue and follow the requirements template.
+
+- Once a every (x) weeks there will be a requirements review meeting where we will approve, reject, or ask for modification to the requirement.
+
+- If approved the requirement will be moved into the design stage in the documents project.
```
Comment (vli11) [inline README.md:11]: If approved, the....

**Review by NeethuES-intel — APPROVED** [review]

**Pull request merged**